FileSystemSimulator/IOSystem/Disk.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include "Cylinder.h"
+class Disk
+{
+public:
+	static const int NUMBER_OF_CYLINDERS = 4;
+	Cylinder** cylinders;
+
+	static const int BLOCK_SIZE = 64;
+	static const int NUMBER_OF_BLOCKS = NUMBER_OF_CYLINDERS * Cylinder::NUMBER_OF_TRACKS * Track::NUMBER_OF_SECTORS;
```

Comment:
does one sector represent logical block in general case?
I believe that sector can contain many blocks (how many? should be defined by a constant)